util.go

```diff
@@ -0,0 +1,73 @@
+package gitbase
```

Comment:
Can we change the name of that file to `path_utils.go` or something like that? I would prefer specific utils that all-in-one utils file with a lot of crap on it.